docs/user/alerting/create-and-manage-rules.asciidoc

```diff
@@ -8,13 +8,21 @@
 The *{stack-manage-app}* > *{rules-ui}* UI provides a cross-app view of alerting.
 Different {kib} apps like {observability-guide}/create-alerts.html[*{observability}*],
 {security-guide}/prebuilt-rules.html[*Security*], <<geo-alerting,*Maps*>> and
-<<xpack-ml,*{ml-app}*>> can offer their own rules. *{rules-ui}* provides a
+<<xpack-ml,*{ml-app}*>> can offer their own rules. 
+
+You can find **Rules** in *Stack Management* > *Alerts and insights* > *Rules* in {kib} or by using
+the <<kibana-navigation-search,global search field>>.
+
+[role="screenshot"]
+image:images/stack-management-rules.png[Rules page navigation]
+
+*{rules-ui}* provides a
 central place to:
 
 * <<create-edit-rules,Create and edit>> rules
 * <<controlling-rules,Manage rules>> including enabling/disabling, muting/unmuting, and deleting
 * Drill down to <<rule-details,rule details>>
-* Configure settings that apply to all rules in the space
+* Configure rule settings
 
 [role="screenshot"]
```

Comment:
> [role="screenshot"]
> image:images/rules-ui.png[Example rule listing in {rules-ui}]

IMO this other screenshot can be removed now

Reply:
I agree! I've deleted it.